docs/build/groups.md

```diff
@@ -0,0 +1,382 @@
+
+```kotlin
+val group = client.conversations.newGroup(listOf(walletAddress1,walletAddress2))
+//Send a message
+group.send("Hello, group!")
+```
+
+</TabItem>
+<TabItem value="swift" label="Swift"  attributes={{className: "swift_tab"}}>
+
+```swift
+let group = try await client.conversations.newGroup(with: [walletAddress1, walletAddress2])
+//Send a message
+try await group.send(content: "Hello, group!")
+```
+
+</TabItem>
+<TabItem value="rn" label="React Native" attributes={{className: "rn_tab "}}>
+
+Code sample coming soon
+
+</TabItem>
+<TabItem value="dart" label="Dart"  attributes={{className: "dart_tab"}}>
+
+Code sample coming soon
+
+</TabItem>
+<TabItem value="js" label="JavaScript"  attributes={{className: "js_tab "}}>
+
+Code sample coming soon
+
+</TabItem>
+<TabItem value="react" label="React"  attributes={{className: "react_tab "}}>
+
+Code sample coming soon
+
+</TabItem>
+</Tabs>
+
+## Manage group chat members
+
+Here are the ways that you manage group chat members with XMTP.
+
+### List group members
+
+Retrieve a list of wallet addresses for all members in the group chat
+
+<Tabs groupId="groupchats">
+<TabItem value="kotlin" label="Kotlin" attributes={{className: "kotlin_tab"}}>
+
+```kotlin
+group.sync()
+val members = group.memberAddresses()
+```
+
+</TabItem>
+<TabItem value="swift" label="Swift"  attributes={{className: "swift_tab"}}>
+
+```swift
+try await group.sync()
+let members = group.members()
+```
+
+</TabItem>
+<TabItem value="rn" label="React Native" attributes={{className: "rn_tab "}}>
+
+Code sample coming soon
+
+</TabItem>
+<TabItem value="dart" label="Dart"  attributes={{className: "dart_tab"}}>
+
+Code sample coming soon
+
+</TabItem>
+<TabItem value="js" label="JavaScript"  attributes={{className: "js_tab "}}>
+
+Code sample coming soon
+
+</TabItem>
+<TabItem value="react" label="React"  attributes={{className: "react_tab "}}>
+
+Code sample coming soon
+
+</TabItem>
+</Tabs>
+
+### Add group members
+
+Add new members to an existing group chat using its wallet address.
+
+<Tabs groupId="groupchats">
+<TabItem value="kotlin" label="Kotlin" attributes={{className: "kotlin_tab"}}>
+
+```kotlin
+group.addMembers(listOf(walletAddress))
+```
+
+</TabItem>
+<TabItem value="swift" label="Swift"  attributes={{className: "swift_tab"}}>
+
+```swift
+try await group.addMembers(addresses: [walletAddress])
+```
+
+</TabItem>
+<TabItem value="rn" label="React Native" attributes={{className: "rn_tab "}}>
+
+Code sample coming soon
+
+</TabItem>
+<TabItem value="dart" label="Dart"  attributes={{className: "dart_tab"}}>
+
+Code sample coming soon
+
+</TabItem>
+<TabItem value="js" label="JavaScript"  attributes={{className: "js_tab "}}>
+
+Code sample coming soon
+
+</TabItem>
+<TabItem value="react" label="React"  attributes={{className: "react_tab "}}>
+
+Code sample coming soon
+
+</TabItem>
+</Tabs>
+
+### Remove group members
+
+Remove a member from an existing group chat using its wallet address
```

Comment:
```suggestion
Remove members from an existing group chat using their wallet addresses
```

This is based on the title of the section being plural and assuming the function takes more than one.